commands/docker-compose.ts

```diff
@@ -77,34 +77,34 @@ async function compose(command: string, message: string, dockerComposeFileUri?:
         const detached: string = configOptions.get('dockerComposeDetached', true) ? '-d' : '';
 
         terminal.sendText(`cd "${folder.uri.fsPath}"`);
-        selectedItems.forEach((item: Item) => {
-            terminal.sendText(command.toLowerCase() === 'up' ? `docker-compose -f ${item.file} ${command} ${detached} ${build}` : `docker-compose -f ${item.file} ${command}`);
-        });
-        terminal.show();
-        if (reporter) {
-            /* __GDPR__
-               "command" : {
-                  "command" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
-               }
-             */
-            reporter.sendTelemetryEvent('command', {
-                command: teleCmdId + command
+        for (let command of commands) {
+            selectedItems.forEach((item: Item) => {
```

Comment:
nit: I know, not your code, but...  personally I don't like using forEach because you can't just debug through the loop with F10, you have to set a breakpoint in the function. for of is so much cleaner now.